aviary/docs/getting_started/onboarding_ext_subsystem.ipynb

```diff
@@ -231,16 +232,18 @@
    "source": [
     "prob = av.AviaryProblem()\n",
     "\n",
-    "prob.load_inputs('models/test_aircraft/aircraft_for_bench_FwFm.csv', phase_info)"
+    "prob.load_inputs('models/test_aircraft/aircraft_for_bench_FwFm.csv',\n",
+    "                 phase_info, meta_data=ExtendedMetaData)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "96ebdf23",
    "metadata": {},
    "source": [
+    "Note that we have set `meta_data` to `ExtendedMetaData` so that we can set battery related variables in the model .csv file.\n",
     "\n",
-    "In the battery subsystem, the type of battery cell we use is `18650`. The option is not set in `input_file`, instead is is controlled by importing the correct battery map [here](https://github.com/OpenMDAO/Aviary/blob/1fca1c03cb2e1d6387442162e8d7dabf83eee197/aviary/examples/external_subsystems/battery/model/reg_thevenin_interp_group.py#L5)."
+    "In the battery subsystem, the type of battery cell we use is `18650`. The option is not set in `input_file`, instead it is controlled by importing the correct battery map [here](https://github.com/OpenMDAO/Aviary/blob/1fca1c03cb2e1d6387442162e8d7dabf83eee197/aviary/examples/external_subsystems/battery/model/reg_thevenin_interp_group.py#L5)."
```

Comment:
Is it still technically an 18650 after changing the discharge rate?

Reply:
I don't know the answer for sure. Here is what John replied to me: "Those cells have varied max discharge rates based on model and application, from about 2-15 A. I'd say changing the upper DV bound and the initial value to 2.0 A makes sense. Really whatever works for that example is good as that is sort of a contrived case."

Reply:
I don't know the answer for sure. Here is what @johnjasa replied to me: "Those cells have varied max discharge rates based on model and application, from about 2-15 A. I'd say changing the upper DV bound and the initial value to 2.0 A makes sense. Really whatever works for that example is good as that is sort of a contrived case."